locales/templates/app.pot

```diff
@@ -1288,6 +1288,12 @@ msgid ""
 "history has been erased."
 msgstr ""
 
+#. Show how many trackers blocked on tracker blocked panel. In front of
+#. "Trackers blocked", show the number of blocking trackers.
```

Comment:
What happens if it's just "1 trackers blocked". Is that use of the plural ok with you?

Reply:
These are my thoughts,
1. For users, to avoid words are beating when reading, I suggest to use plural nouns for all cases.
2. I referred to Firefox Focus which has the same feature in its menu when browsing a website. It use plural nouns (Trackers blocked) for 0 or 1 case as well.

So I think use of the plural should be ok.

Reply:
ok thanks!